schemas/org.gnome.shell.extensions.org-scrollwm.gschema.xml

```diff
@@ -172,6 +172,26 @@
         <summary>Expel the bottom window into its own column</summary>
     </key>
 
+    <key type="as" name="resize-h-inc">
+      <default><![CDATA[['<Super><Shift>equal']]]></default>
+      <summary>Increment window height</summary>
+    </key>
+
+    <key type="as" name="resize-h-dec">
+      <default><![CDATA[['<Super><Shift>minus']]]></default>
+      <summary>Decrement window height</summary>
+    </key>
+
+    <key type="as" name="resize-w-inc">
+      <default><![CDATA[['<Super>equal']]]></default>
```

Comment:
We want to have `plus` too here, as there's layouts with unshifted plus.

Reply:
okay. but then it's the same keybind as resize-h-inc for e.g. US layouts with plus being <Shift>equal. or am i missing something?

Reply:
Ah, right, that's a problem.

Reply:
I mean we could go for ctrl instead of shift and it would be okay. Maybe not nice to reach, but without conflicts and then everyone can change their bindings.

Reply:
Did a quick test, seems to work correctly. Mutter doesn't care about shifting/case eg. `<super>r` and `<super>R` is the same keybinding.

Reply:
Actually, it looks like `<super>plus` also works in us layout. Ie. `<super>plus` and `<super>equal` is the same binding in us layout.

Reply:
ah nice. okay, then we'll go for that.